extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTenantConfigResolver.java

```diff
@@ -119,10 +117,10 @@ public OidcTenantConfig apply(OidcTenantConfig tenantConfig) {
                             final String tenantId = context.get(OidcUtils.TENANT_ID_ATTRIBUTE);
 
                             if (tenantId != null && !isTenantSetByAnnotation(context, tenantId)) {
-                                TenantConfigContext tenantContext = tenantConfigBean.getDynamicTenantsConfig().get(tenantId);
+                                TenantConfigContext tenantContext = tenantConfigBean.getExistingTenantConfigContext(tenantId);
```

Comment:
`Existing` includes both static and dynamic ones, so propose to keep the original name for now